### DISCUSSION TITLE
Add test for issue #1484

### DISCUSSION THREAD

**Discussion by openandclose** [pymupdf/PyMuPDF#1508] [Q&A] [answered]
Thank you for fixing #1484, and adding API for TrimBox etc.

How about adding test for box coordinates (since I had to check and already had written)?

(test_pagebox branch)
https://github.com/openandclose/PyMuPDF/commit/a9daf21aec403063a2b697c46ca8c99d79275233

**Comment by JorjMcKie** [comment, marked as answer]
Yes, a good idea. Thank you.

**Comment by openandclose** [comment]
Oh I remembered GPL conplicaitons.
Don't use my code. I'm fine.

**Comment by openandclose** [comment]
Personally, I thought this was precisely the subject to add tests preferentially,
and this was a good occasion to do so
(even without my kind-of pull request, that is, with your code).

Do you have a different plan?
Then I'll have to close this discussion.

**Reply by JorjMcKie** [reply]
You can close it: the next version 1.19.5 will have this type of test.
